yarelismartin/skill-swap
# Posts/ym

## Description
I made it so the user is now able to create an dupdate their post by filling out and submitting the post form. 
## Related Issue
#4 

## Motivation and Context
This is required so the user can contribute their thoughts to the community page. 
## How Can This Be Tested?
In the community page there is a button labeled 'create post' when this is clicked it will route you to the post form. When the form if filled out and submitted the user can then see their post among others. Their post will have the 3 dot more option icon that gives the user the option to edit their post. 
## Screenshots (if appropriate):

## Types of changes

[ ] Bug fix (non-breaking change which fixes an issue)
[x ] New feature (non-breaking change which adds functionality)
[ ] Breaking change
